testgrid/config/config.yaml

```diff
@@ -1925,6 +1925,15 @@ dashboards:
   dashboard_tab:
     - name: daisy-e2e
       test_group_name: ci-daisy-e2e
+      open_test_template:
+        url: https://storage.googleapis.com/<gcs_prefix>/<changelist>/artifacts/log-<test-id>.txt
```

Comment:
@michelle192837 is <test-id> a thing?

Reply:
<test-id> is; currently it's a hash of the test path, I just added code that will be pushed next week or so that will recognize an 'id' attribute as an invocation ID as well.

So right now this will link to nothing useable (because test-id doesn't match what's expected), but once the code's pushed it should link to the correct value if 'id' is present as an attribute in the test data.